sycl/include/sycl/ext/oneapi/matrix/matrix-jit-use.hpp

```diff
@@ -256,6 +257,21 @@ class wi_element {
   wi_element(joint_matrix<T, NumRows, NumCols, Use, Layout, Group> &Mat,
              std::size_t i)
       : M(Mat), idx(i) {}
+
+  std::tuple<size_t, size_t> get_coord() {
+#ifdef __SYCL_DEVICE_ONLY__
+    __ocl_vec_t<int32_t, 2> co_ord =
```

Comment:
remove underscore from the name (co_ord)

Reply:
Don't see it's applied